web/firefox_print_service.js

```diff
@@ -0,0 +1,120 @@
+/* Copyright 2016 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+(function (root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define('pdfjs-web/firefox_print_service', ['exports', 'pdfjs-web/ui_utils',
+      'pdfjs-web/app', 'pdfjs-web/pdfjs'], factory);
+  } else if (typeof exports !== 'undefined') {
+    factory(exports, require('./ui_utils.js'), require('./app.js'),
+      require('./pdfjs.js'));
+  } else {
+    factory((root.pdfjsWebFirefoxPrintService = {}), root.pdfjsWebUIUtils,
+      root.pdfjsWebApp, root.pdfjsWebPDFJS);
+  }
+}(this, function (exports, uiUtils, app, pdfjsLib) {
+  var CSS_UNITS = uiUtils.CSS_UNITS;
+  var PDFPrintServiceFactory = app.PDFPrintServiceFactory;
+
+  function composePage(pdfDocument, pageNumber, size, printContainer) {
```

Comment:
This is called `renderPage` in `pdf_print_service.js`. Can we name it `renderPage` here too for consistency?